### PR TITLE
Hello microverse

### PR DESCRIPTION
**Pull request by fatima-najafi**
In this project i implement the following steps:

- Create a new GitHub repo.
- Create a new branch according to the [GitHub flow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md) rules.
- Add a .gitignore file.
- Add node_modules to gitignore file.
- Set up a linter for HTML and CSS
- Add descriptive README file to your project 
- Add 3 files (each one in a separate commit):

- Open a pull request.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,167 @@
...
+
+I would like to thank every who continues supporting me.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT.md) licensed.
```
Comment (V-Blaze) [inline README.md:165]: - [ ] You have done exceptionally great, You just need to update a broken link before moving to the next milestone. Currently the MIT license link in your readme file leads to a 404 error page. A broken link in a readme file makes it look unprofessional, I'll suggest you create a `MIT.md` file and add the license content, You can [use this ](https://github.com/V-Blaze/gvash-planes/blob/dev/LICENSE)and then customize it for your project. :+1: :smile: 

- - 
Screenshot from 2023-05-30 13-47-04

**Review by V-Blaze — CHANGES_REQUESTED** [review]
## Hi @fatima-najafi  , 

## Required changes 🖊️ 
# Almost there 🟡 🟦 🟨 

giphy


Great work so far, You have come a long way, keep moving forward👏 👍  
You've done well implementing the project requirements, but there are still some issues I think you need to improve on. 😄 👋 


 ## Highlights
- Correct GitHub flow✔
-  Linters setup correctly👋 
-  Used correct readme template✔

 ## Required Changes ♻️

## Optional suggestions
- [OPTIONAL] Your commit messages are good but following the best practices, you are meant to capitalize the first letter of the first word in the message, I'll suggest you put this into practice in your subsequent projects :+1: :+1: 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by fatima-najafi** [comment]
hi sir , thank you for reviewing my code i make changes please can you
check it again, is it right ?

File: README.md [hunk 2]
```diff
@@ -1 +1,167 @@
...
+
+- **Display colorfull text**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Demo link is coming soon](#)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (roshan-bajgain) [inline README.md:73]: - [ ] [Optional] I would suggest you please remove this live demo link if you don't deploy this app.

**Review by roshan-bajgain — APPROVED** [review]
Hi @fatima-najafi, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**